### ISSUE TITLE
2-2-3　p.78中部に関して

### ISSUE THREAD

**Issue by Semayuki** [YutaroOgawa/pytorch_advanced#207]
`imgs=torch.stack(imgs,dim=0)`について、invalid syntaxのエラーが表示されてしまいます。
そこまでのコードは本の内容と完全に同じになっています。

imgsがtensor化していないのかと思い、あれこれやってはみましたが無駄のようです。
colabの使用が原因かもしれません。

対策をご教授いただければと思います。
よろしくお願い致します。

**Comment by YutaroOgawa** [comment]
@Semayuki さま

こちら、深い調査が必要となるかもしれず、現在、そこに対応する余裕がございません。
しばらくお時間がかかります。
大変申し訳ございませんが、どうぞよろしくお願い致します。

他の読者の方で、同様の現象に陥っている方がいましたら、状況を教えていただけると幸いです。

どうぞよろしくお願い致します。

**Comment by siruku6** [comment]
2021/11/25発行の第9刷 p.78のソースコード（[2-3 DataLoaderの実装]中の `od_collate_fn`内)

> imgs = torch.stack(imgs, dim=0)

については、問題なく動作しました。

版・刷と、実際に記載されていたソースコード全体の情報がないとなんとも言えないのですが、
今回のエラーが `invalid syntax` だったとすると、
データの誤りなどではなくて、Pythonの文法上の誤りだったのではないかと思います。

ご参考まで。
